index.html

```diff
@@ -1,67 +1,324 @@
+              <h1>About <br>
+              Myself</h1>
+            </div>
+            <div class="primary-text">
+              <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+            <div class="social-media">
+              <div class="supporting-text">
+                <span>LET'S CONNECT!</span>
+              </div>
+            </div>
+            <div class="social-media-icons">
+              <ul class="social-media-icons-list">
+                <li><a href="#"><img src="assets/twitter.svg" alt="Twitter" width="20" height="20"></a></li>
+                <li><a href="#"><img src="assets/linkedin.svg" alt="Linkedin" width="20" height="20"></a></li>
+                <li><a href="#"><img src="assets/m.svg" alt="Medium" width="20" height="16"></a></li>
+                <li><a href="#"><img src="assets/git.svg" alt="GitHub" width="18" height="20"></a></li>
+                <li><a href="#"><img src="assets/finger.svg" alt="Peace" width="15" height="20"></a></li>
+              </ul>
+            </div>
+            <div class="normal-button-secondary">
+              <div class="get-my-resume">
+                <span>Get my resume</span>
+              </div>
+            </div>
+          </div>
+          <div class="skill-list">
+            <div class="skill-languages">
+              <div class="languages">
+                <span>
+                  Languages
+                </span>
+              </div>
+              <div class="disabled-languages">
+                <img class="disabled" src="assets/Disabled.svg" alt="disabled">
+              </div>
+            </div>
+            <div class="languages-list">
+              <ul class="social-media-icons-list">
+                <li><a href="#"><img src="assets/frame-9.svg" alt="Twitter" width="303" height="66"></a></li>
+                <li><a href="#"><img src="assets/frame-10.svg" alt="Linkedin" width="303" height="66"></a></li>
+                <li><a href="#"><img src="assets/frame-11.svg" alt="Medium" width="303" height="66"></a></li>
+                </ul>
+            </div>
+            <div class="skill-frameworks">
+              <div class="frameworks">
+                <span>
+                  Frameworks
+                </span>
+              </div>
+              <div class="chevron">
+                <img class="chevron" src="assets/chevron.svg"  alt="chevron">
+              </div>
+            </div>
+            <hr class="solid">
+            <div class="skill-skills">
+              <div class="skills">
+                <span>
+                  Skills
+                </span>
+              </div>
+              <div class="chevron">
+                <img class="chevron" src="assets/chevron.svg" alt="chevron">
+              </div>
+            </div>
```

Comment:
- [ ] Kindly make use of the `<ul>` and `<li>` tags to wrap the list of technologies. e.g.
``` javascript
  <ul>
     <li>.....</li>
  <ul>
```